proposals/nnnn-mt-registration-limitation.md

```diff
@@ -1,133 +1,122 @@
-; If same AppHMIType is specified in both AcceptedAppHMITypeTable and DisabledAppHMITypeTable, the entry in DisabledAppHMITypeTable is
-; ignored and apps with the AppHMIType are launched as usual.
-; also, if an app has multiple AppHMITypes, and they are listed in both AcceptedAppHMITypeTable and DisabledAppHMITypeTable, then the
-; app is launched as usual.
-; If an AppHMIType is not specified to neither AcceptedAppHMITypeTable nor DisabledAppHMITypeTable, then apps with the AppHMIType
-; are rejected for registration through the transport. WiFi-iOS = NAVIGATION, PROJECTION
-WiFi-Android = none
-iAP = none
-Bluetooth-SPP = none
-AOA = none
+[TransportRequiredForResumption]
+; This section specifies transport types that are required to trigger resumption for each AppHMIType.
+; App has to be connected through at least one of the transport listed (either as the Primary
+; Transport or Secondary Transport) to trigger resumption. If the app is not connected with any of
+; the transports listed, its HMIlevel will be kept in NONE and the state stays in NOT_AUDIBLE.
+; In case an app has multiple AppHMIType, requirements of all of the AppHMITypes are applied.
+; Possible AppHMITypes: DEFAULT, COMMUNICATION, MEDIA, MESSAGING, NAVIGATION, INFORMATION, SOCIAL,
+;                       BACKGROUND_PROCESS, TESTING, SYSTEM, PROJECTION, REMOTE_CONTROL, NONE
+; Possible transport types: TCP_WIFI, IAP_CARPLAY, IAP_USB_HOST_MODE, IAP_USB_DEVICE_MODE, IAP_USB,
+;                           AOA_USB, IAP_BLUETOOTH, SPP_BLUETOOTH
+;
+; The default behavior is to always enable resumption. If an AppHMIType is not listed in this
+: section, resumption is enabled for an app with the AppHMIType.
+; On the other hand, if you want to disable resumption and always keep an app in NONE and
+; NOT_AUDIBLE state after registration, specify an empty value for the AppHMIType.
+
+; DEFAULT =
+; COMMUNICATION =
+; MEDIA =
+; MESSAGING =
+NAVIGATION = TCP_WIFI, IAP_CARPLAY, IAP_USB_HOST_MODE, IAP_USB_DEVICE_MODE, IAP_USB, AOA_USB
+; INFORMATION =
+; SOCIAL =
+; BACKGROUND_PROCESS =
+; TESTING =
+; SYSTEM =
+PROJECTION = TCP_WIFI, IAP_CARPLAY, IAP_USB_HOST_MODE, IAP_USB_DEVICE_MODE, IAP_USB, AOA_USB
+; REMOTE_CONTROL =
+; "NONE" applies to apps that don't specify any AppHMIType
+; NONE =
 ```
 
-In these sections, `WiFi-iOS` entry is for registration from iOS app through Wi-Fi transport. `WiFi-Android` is for registration from Android app through Wi-Fi transport. `iAP` is for iOS apps. `Bluetooth-SPP` and `AOA` are for Android apps.
+The example shown in the smartDeviceLink.ini above indicates that `NAVIGATION` and `PROJECTION` apps require one of Wi-Fi or USB transports to enable resumption. Which means, when they are connected only with Bluetooth transport, their HMILevels are kept in NONE and they are put in NOT\_AUDIBLE state. Other AppHMITypes do not have restriction, so resumption will be always enabled for them.

```

Comment:
Please update the first sentence to "...apps require either Wi-Fi or USB transports..."  Please update the last sentence to "...resumption will always be enabled for them."